index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    
+    <section id = "welcome-section"> 
+      <h1> Hi in this section your are going to view my portfolio </h1>
+       </section>>
```

Comment:
I think it would be better if you use <header> here. Remember always using semantic tags will help your code be cleaner.

Reply:
- [ ]  Nice effort pointing out the use of `semantic` tags. However, it is good to be clear as much as possible on what the issue is, and also back up your suggestions with reasons why you think they should do so. Please, let us edit this comment to be clear on what is needed to be changed and give a reason to as well. It makes the developer feel appreciated for their effort and they won't regard changes as negative feedback.